web-app-sql-docdb-search/README.md

```diff
@@ -0,0 +1 @@
+# Provision a web app with a SQL Database, DocumentDB and Azure Search

```

Comment:
Hi!
Since it's not yet available in the repo, can I still generate the buttons or do I need to wait until I've cleared the CLA part?